tuna.go

```diff
@@ -465,13 +490,80 @@ func (c *Common) CreateServerConn(force bool) error {
 				allSubscribers[i], allSubscribers[j] = allSubscribers[j], allSubscribers[i]
 			})
 
+			// filter
+			var filterSubscribers filterSubscribers
```

Comment:
It's better to change variable name to be different from variable type name `filterSubscribers`

Reply:
Also you can use `make(filterSubscribers, 0, len(allSubscribers))` to set a cap in advance to avoid re-alloc memory during append. It might waste some memory but will reduce the number of memory allocations